Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -183,13 +188,16 @@ class TopTabsViewController: UIViewController {
     }
     
     func scrollToCurrentTab(animated: Bool = true) {
+        if !animated {
+            collectionView.contentOffset = CGPoint.zero
```

Comment:
if the normal tab mode (with a bunch of tabs) was scrolled all the way to the right and then say private mode has 2 tabs, the scrollRectToVisible would look really odd like: 
![simulator](https://cloud.githubusercontent.com/assets/11801330/16594396/3e137b5a-42b8-11e6-92c5-54996553f17b.png)

But switching it to scrollToItemAtIndexPath also fixed this issue so I removed it :)